src/Microsoft.ML.OnnxConverter/OnnxUtils.cs

```diff
@@ -376,7 +376,7 @@ public ModelArgs(string name, TensorProto.Types.DataType dataType, List<long> di
                 }
             }
             //batch size.
-            dimsLocal?.Insert(0, 1);
+            dimsLocal?.Insert(0, -1);
```

Comment:
>dimsLocal?.Insert(0, -1); [](start = 12, length = 25)

PLs add comment describing this value -1 with reference to original ONNX doc on web

Reply:
added more comments.